includes/Dashboard.php

```diff
@@ -103,15 +103,38 @@ public function get_hook_suffix() {
 	 * @return void
 	 */
 	public function add_menu_page() {
+		global $submenu, $menu;
+		$link = 'edit.php?post_type=' . Story_Post_Type::POST_TYPE_SLUG;
+
 		$this->hook_suffix = add_submenu_page(
-			'edit.php?post_type=' . Story_Post_Type::POST_TYPE_SLUG,
-			__( 'Dashboard', 'web-stories' ),
+			$link,
 			__( 'Dashboard', 'web-stories' ),
+			__( 'My Stories', 'web-stories' ),
 			'edit_web-stories',
 			'stories-dashboard',
 			[ $this, 'render' ],
 			0
 		);
+
+		add_submenu_page(
+			$link,
+			__( 'Explore templates', 'web-stories' ),
+			__( 'Explore templates', 'web-stories' ),
```

Comment:
```suggestion
			__( 'Explore Templates', 'web-stories' ),
			__( 'Explore Templates', 'web-stories' ),
```

Let's use this version to match what users see on the dashboard and re-use an existing string.

(Yes I know #2750 says `Explore templates`)